.circleci/config.yml

```diff
@@ -59,3 +69,11 @@ jobs:
       - store_artifacts:
           path: coverage
           destination: ~/coverage
+
+workflows:
+  build_and_test:
+    jobs:
+      - build:
+          ruby_version: 2.3.8
+      - build:
+          ruby_version: 2.5.5
```

Comment:
Your probably on this already but do you want to add 2.6.2 to this one as well?

Reply:
I just added 2.6.2.